src/arm/BBORG_SERVO-00A2.dts

```diff
@@ -75,45 +75,27 @@
     #address-cells = <1>;
     #size-cells = <0>;
 
-    pca9685_servo: pca9685_serve@7f {
-        compatible = "nxp, pca9685";
+    pca: pca@70 {
+        compatible = "nxp,pca9685-pwm";
+        #pwm-cells = <16>;
+        reg = <0x70>;
         lable = "pca9685_servo";
-        pinctrl-names = "default";
-        pinctrl-0 = <&bborg_servo_pca9685_en_pins>;
-        enable-gpios = <&gpio2 4 GPIO_ACTIVE_HIGH>;
```

Comment:
Is there or is there not an enable gpio? I'd think you'd need that.

Reply:
There is but we handle it separately not, I tried using it this way but I was not able to make it work out.